### PR TITLE
Add Default env variables on CES V2 Orchestration Dockerfile

### LINKED ISSUE #19: Add Env Variables on CES V2 Orchestration Dockerfile
The Dockerfile should next default ENV variables:

- CDS_HOST_URL
- SLS_HOST_UR
- HAPI_FHIR_URL

### PR DESCRIPTION
**Pull request by sgroh**

### CODE REVIEW

**Comment by mojitoj** [comment]
@sgroh preferably we should not hard-code these values here in the Dockerfile. Is it possible to read these env variables from the cloudbuild environment so that they are not hard-coded in the repo?

File: leap-ces-v2-orchestration/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,10 @@
 FROM openjdk:11-jdk
 
 ARG JAR_FILE=target/leap-ces-v2-orchestration-0.0.1.war
+
+ENV CDS_HOST_URL=https://sdhc-leap.appspot.com
```
Comment (mojitoj) [inline leap-ces-v2-orchestration/Dockerfile]: as commented on the PR

**Review by mojitoj — CHANGES_REQUESTED** [review]

**Comment by sgroh** [comment]
> @sgroh preferably we should not hard-code these values here in the Dockerfile. Is it possible to read these env variables from the cloudbuild environment so that they are not hard-coded in the repo?

We always have the option to override them calling the docker with such environment variables when we run the container i.e: 
`docker run -e "CDS_HOST_URL=new_URL" `
BTW, if we want to remove this from the repository the solution should take the longest path, because when you select `Build Configuration: Dockerfile` on the trigger there is no option to add custom environment variables. In order to do that is necessary to specify `Cloud Build Configuration file (yaml or json)` option and add a couple of manual steps inside a new cloudbuild.yaml file. This last step is not the default way to resolve the CD but we can try this if having env variables inside a Dockerfile is not an option here. IMO have default values for mandatory env variables should be a must, but having them externalized is a good nice to have.

**Comment by mojitoj** [comment]
I think it's just not good practice to have specific deployment configurations like this hard coded in the repo. What I had in mind was to do something like this:
```
environment
  CDS_HOST_URL =${_CDS_HOST_URL}
```
in which the right-hand side is an env variable injected by build trigger at build time and can be defined on the build trigger.

**Review by ddecouteau — APPROVED** [review]

**Review by mojitoj — APPROVED** [review]

**Pull request merged**